### DISCUSSION TITLE
iOS app can’t change language

### DISCUSSION THREAD

**Discussion by Shinji-IkariG** [community/community#142919] [Mobile]
There's no language setting in my GitHub setting (iOS config page)

![image](https://github.com/user-attachments/assets/24735870-e8ef-435f-98a3-505b157952d5)
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.183.1 (145298821)
Device: iPhone 13 (iOS 17.7)
</details>

**Comment by qinscode** [comment]
This function has not been implemented yet.

**Comment by Iron-Ham** [comment]
Hi @Shinji-IkariG, 

Thanks for reaching out! I'm happy to help you with this. The language selection option appears under the GitHub app settings only if you have more than one language set in your phone's general settings. Can you check to see if you have that enabled on your device by following these steps:

- Visit the iOS Settings App
- Tap "General"
- Tap "Language & Region"
- Ensure you have more than one language in that list. 

I realize that these settings may be named differently for you depending on your device's locale, but I've attached a video demonstration as well:

<details><summary>Video Demonstration</summary>
<p>


https://github.com/user-attachments/assets/0351271b-18dd-4481-ab2a-6cb037f907a5



</p>
</details> 

Once you set a secondary language in your device settings, the GitHub app's settings will look like this:

| With One Language Set | With Two Languages Set | Language Options |
|--------|--------|-------|
| ![IMG_1426](https://github.com/user-attachments/assets/fa7c290e-43e0-42ce-959a-c1b8d53e11a3) | ![IMG_52B2DFEC359C-1](https://github.com/user-attachments/assets/c1ae427e-6bd8-4dc1-9d53-24dbae0e9a85)  | ![IMG_1423](https://github.com/user-attachments/assets/da7cb59d-8552-4fd3-8e0d-a8f8f79c0d02) |

**Reply by Shinji-IkariG** [reply]
My friends all have the option to adjust the language, but mine doesn't. My iOS app is already the latest version (and I've tried reinstalling it).
